<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>352</width>
     <height>0</height>
    </size>
   </property>
   <widget class="MyOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>361</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewScene"/>
   <addaction name="actionOpenScene"/>
   <addaction name="actionSaveScene"/>
   <addaction name="separator"/>
   <addaction name="actionReadme"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="hierarchy">
   <property name="minimumSize">
    <size>
     <width>94</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="inspectorDock">
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="renderingDock">
   <property name="windowTitle">
    <string>Rendering</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="project">
   <property name="windowTitle">
    <string>Project</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewScene"/>
    <addaction name="actionOpenScene"/>
    <addaction name="actionSaveScene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReadme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenScene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Folder.png</normaloff>:/icons/Resources/icons/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open scene</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Disquette.png</normaloff>:/icons/Resources/icons/Disquette.png</iconset>
   </property>
   <property name="text">
    <string>Save scene</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNewScene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Document-new.png</normaloff>:/icons/Resources/icons/Document-new.png</iconset>
   </property>
   <property name="text">
    <string>New scene</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Info.png</normaloff>:/icons/Resources/icons/Info.png</iconset>
   </property>
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
